.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: golang:1.12.7
+      - image: circleci/golang:1.12.7
```

Comment:
Do you want to float? Like `golang:1.12`?

Reply:
yea good idea. will update thnx